lib/ts/recipe/accountlinking/utils.ts

```diff
@@ -42,3 +45,72 @@ export function validateAndNormaliseUserInput(_: NormalisedAppinfo, config?: Typ
         shouldDoAutomaticAccountLinking,
     };
 }
+
+export async function verifyEmailForRecipeUserIfLinkedAccountsAreVerified(input: {
```

Comment:
Had previously moved this because of the duplicated function in mfa recipe. This move is no more required, so have reverted this. There are no logical changes.